lib/mindwendel_web/live/idea_live/form_component.ex

```diff
@@ -48,12 +48,15 @@ defmodule MindwendelWeb.IdeaLive.FormComponent do
   end
 
   defp save_idea(socket, :new, idea_params) do
-    Mindwendel.Accounts.update_user(socket.assigns.current_user, %{
-      username: idea_params["username"]
-    })
+    {:ok, user} =
+      Mindwendel.Accounts.update_user(socket.assigns.current_user, %{
+        username: idea_params["username"]
```

Comment:
maybe we only want to trigger this if the user name actually changed? Although I'm unsure maybe ecto is intelligent enough to not issue a query if it didn't change.

Reply:
ok good, as I thought it's intelligent enought :)

```
iex(5)> user.username
"No more Fred"
iex(6)> Accounts.update_user(user, %{username: "No more Fred"})
{:ok,
 %Mindwendel.Accounts.User{
   __meta__: #Ecto.Schema.Metadata<:loaded, "users">,
   id: "cd21937f-d428-4deb-96e5-bf7e7559817b",
   username: "No more Fred",
   created_brainstormings: #Ecto.Association.NotLoaded<association :created_brainstormings is not loaded>,
   brainstormings: #Ecto.Association.NotLoaded<association :brainstormings is not loaded>,
   moderated_brainstormings: #Ecto.Association.NotLoaded<association :moderated_brainstormings is not loaded>,
   ideas: #Ecto.Association.NotLoaded<association :ideas is not loaded>,
   inserted_at: ~N[2024-09-15 08:38:19],
   updated_at: ~N[2024-10-04 06:15:59]
 }}
```